CHANGELOG.md

```diff
@@ -2,6 +2,87 @@
 
 Notable changes to QMCPACK are documented in this file.
 
+##  [3.15.0] - 2022-09-29
+
+This is a recommended release for all users. There are many quality of life
+improvements, bugfixes throughout the application, and updates to the associated
+testing. Thanks to everyone who reported an issue or suggested an improvement.
+
+We are working to make the performance portable "batched drivers" the default in
+an upcoming version. These support execution on CPUs and multiple GPU
+architectures with high performance. Most standard QMC calculations and many
+observables are already supported. Because some changes to the input files will
+be required, we recommend trying these drivers now and reporting any issues.
+
+* Important bug fix to excited states in splines when spin-up/down sets are
+  built from the same spin species and occupation is specified on the first sposet
+  [\#4158](https://github.com/QMCPACK/qmcpack/pull/4158)
+* The Quantum ESPRESSO converter, pw2qmcpack is now supported via a plugin
+  activated via -DQE_ENABLE_PLUGINS=pw2qmcpack on the QE CMake configure line,
+  see
+  https://qmcpack.readthedocs.io/en/develop/installation.html#quantum-espresso-7-0
+  . The latest QE 7.1 and earlier 7.0 are supported, and new versions should be
+  automatically compatible.
+* Substantial improvements to the performance portable / batched implementation.
+  Using LLVM 15.0, high performance production calculations can be performed on
+  NVIDIA GPUs for several wavefunction types, in addition to running on all CPU
+  systems. As introduced in v3.14.0, the optional project data input parameter
```

Comment:
The "project data input parameter" should be a separate bullet point?

Reply:
Or nested bullet points for llvm 15 and driver input parameter?

Reply:
Good suggestion. I have now split this part into two bullet points. More readable